<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1079</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 12pt &quot;Arial&quot;;</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>500</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>目标速度</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>440</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>初始速度</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_startspeed">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>440</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_targetspeed">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>500</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_start">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>610</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>启动</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>440</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>P：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>500</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>I：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>560</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>D：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_p">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>440</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_i">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>500</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_d">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>560</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>560</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>当前速度</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_curspeed">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>565</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_reset">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>610</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>复位</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>1000</width>
     <height>400</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_stop">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>610</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>停止</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_1">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>440</y>
     <width>110</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>测试1</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_2">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>490</y>
     <width>110</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>测试2</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_3">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>540</y>
     <width>110</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>测试3</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_4">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>590</y>
     <width>110</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>测试4</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
